manticore/core/smtlib/solver.py

```diff
@@ -129,7 +130,8 @@ def __init__(self):
         super().__init__()
         self._proc = None
 
-        self._command = 'z3 -t:240000 -memory:16384 -smt2 -in'
+        self._command = \
+            f'z3 -t:{settings.getint("settings", "solver_timeout", fallback=240000)} -memory:16384 -smt2 -in'
```

Comment:
If settings.solver_timeout was not defined and the fallback was used .. maybe it should be recorded in the .ini that will go to the workspace.